### PR TITLE
chore: Replace Spinner's SVG animation with an animated conic-gradient background

### PR DESCRIPTION
**Pull request by behowell**
## Related Issue(s)

- Fixes #

### CODE REVIEW

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30351/perf-test-react-components/Avatar-mount_base.html">628</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30351/perf-test-react-components/Avatar-mount.html">644</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30351/perf-test-react-components/Button-mount_base.html">308</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30351/perf-test-react-components/Button-mount.html">305</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30351/perf-test-react-components/Field-mount_base.html">1130</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30351/perf-test-react-components/Field-mount.html">1147</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30351/perf-test-react-components/FluentProvider-mount_base.html">726</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30351/perf-test-react-components/FluentProvider-mount.html">709</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30351/perf-test-react-components/FluentProviderWithTheme-mount_base.html">82</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30351/perf-test-react-components/FluentProviderWithTheme-mount.html">84</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30351/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">71</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30351/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">60</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30351/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">74</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30351/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">80</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30351/perf-test-react-components/MakeStyles-mount_base.html">867</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30351/perf-test-react-components/MakeStyles-mount.html">897</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30351/perf-test-react-components/Persona-mount_base.html">1785</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30351/perf-test-react-components/Persona-mount.html">1697</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30351/perf-test-react-components/SpinButton-mount_base.html">1366</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30351/perf-test-react-components/SpinButton-mount.html">1366</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports                                                                             |  Baseline (minified/GZIP) |                          PR |                                                                                                                                                                                                        Change |
| :-------------------------------------------------------------------------------------------- | ------------------------: | --------------------------: | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
| <samp>react-spinner</samp> <br /> <abbr title='bundle-size/Spinner.fixture.js'>Spinner</abbr> | `23.4 kB`<br />`8.467 kB` | `29.67 kB`<br />`10.818 kB` | `6.27 kB` <img aria-hidden="true" src="https://microsoft.github.io/monosize/images/increase.png" /><br />`2.351 kB` <img aria-hidden="true" src="https://microsoft.github.io/monosize/images/increase.png" /> |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr>              |  `69.901 kB`<br />`20.261 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `210.494 kB`<br />`60.055 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `42.388 kB`<br />`14.103 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |    `7.099 kB`<br />`2.385 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/ae31ae01c299e43ff98c561bf937da1ed465ef8a'>ae31ae01c299e43ff98c561bf937da1ed465ef8a</a></sub>
<div id="bundleSizeComment9423"/>

File: change/@fluentui-react-spinner-65e1fe2a-ca72-408e-90eb-6a1ca3f6caca.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-spinner-65e1fe2a-ca72-408e-90eb-6a1ca3f6caca.json:1]: ## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30351&commitId=46bbdffd528c0db56faf555dd12283dafe84f566&env=prod&clientType=FLUENTUI)


  <details><summary><b>Spinner converged</b> 14 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Spinner converged.Primary with Label - Dark Mode.chromium.png|287| Changed|
|Spinner converged.Inverted with Label - High Contrast.chromium.png|883| Changed|
|Spinner converged.Inverted with Label - Dark Mode.chromium.png|467| Changed|
|Spinner converged.Inverted - High Contrast.chromium.png|221| Changed|
|Spinner converged.Primary - Dark Mode.chromium.png|72| Changed|
|Spinner converged.Inverted.chromium.png|117| Changed|
|Spinner converged.Primary + size.chromium.png|767| Changed|
|Spinner converged.Inverted - Dark Mode.chromium.png|117| Changed|
|Spinner converged.Primary with Label - High Contrast.chromium.png|868| Changed|
|Spinner converged.Primary with Label - RTL.chromium.png|426| Changed|
|Spinner converged.Inverted with Label.chromium.png|467| Changed|
|Spinner converged.Primary.chromium.png|106| Changed|
|Spinner converged.Primary - High Contrast.chromium.png|217| Changed|
|Spinner converged.Primary with Label.chromium.png|426| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

**Comment by behowell** [comment]
It appears that this does not solve performance problems with the svg animation Edge/Chrome. The animated CSS variables seem to use the main thread to animate. Additionally, animated CSS variables are not yet supported by Firefox.

**Pull request closed without merging**